homework.md

```diff
@@ -4,77 +4,77 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Mark-up Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Styling a web page
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+.h1 {
 }
 ```
```

Comment:
HI Kudah your h1 is targeting h1 class selector with that period in front which is not correct.

Reply:
Hey Jontas, thanks for letting me know but I am a little bit stuck could you please explain for me.

Reply:
Hi Kudah in HTML if we want to target all the elements within the <body> we would do it like this in CSS 

body {      
}

but if we want to target a specific element we create a class attribute for ex. <div class="container"> we would then target the container like this in CSS

.container {     /* note the period in front */
}

This is just short explanation hope you understand if not just give me a shout I will gladly assist you further.